pkg/helm/chart.go

```diff
@@ -108,6 +108,7 @@ func overwriteValuesFile(chartDir string, values helmify.Values) error {
 	}
 
 	file := filepath.Join(chartDir, "values.yaml")
+	res = append(res, []byte("cert-manager:\n  enabled: false\n  installCRDs: true")...)
```

Comment:
- check certManagerAsSubchart flag
- use Add method to add values to `values helmify.Values`